android/src/main/AndroidManifest.xml

```diff
@@ -1,10 +1,6 @@
-
-  <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="com.getcapacitor.community.intercom.intercom">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.getcapacitor.community.intercom.intercom">
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.MANAGE_DOCUMENTS"/>
     <uses-permission android:name="android.permission.VIBRATE"/>
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
These did not appear to be needed from what I could tell in the Intercom SDK documentation but wanted to point this out to confirm.